### ISSUE TITLE
serilog-sinks-http 9.0.0 release causes issues on startup

### ISSUE THREAD

**Issue by eozdil174** [BetterStackHQ/logs-client-serilog#5]
Hello BetterStack team, 

while updating my application packages, I noticed on startup I was getting the following error: 

```
Method 'PostAsync' in type 'BetterStack.Logs.Serilog.BetterStackHttpClient' from assembly 'BetterStack.Logs.Serilog, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

Looking at the [last release](https://github.com/FantasticFiasco/serilog-sinks-http/releases/tag/v9.0.0) for the serilog-sinks-http. The PostAsync method was changed (a new parameter was added), which seems to be the cause of the problem.

It would be great if this could be updated so people don't have to dig to realise the issue :)

**Comment by curusarn** [comment]
Hey @eozdil174!

Thank you for raising this.
I'm passing this on to the team to get this fixed.

Thanks again for bringing this up!

**Comment by PetrHeinz** [comment]
Hi @eozdil174,

We've just released `v1.1.0` of our `BetterStack.Logs.Serilog` library that is compatible with `Serilog.Sinks.Http` in `v9.0.0`, and it should be available on NuGet shortly.

Thanks again for reporting this 🙌 

Have an awesome day!
